tests/ui/UnreadIndicatorsTest.tsx

```diff
@@ -143,43 +154,33 @@ function scrollUpToRevealNewMessagesBadge() {
     });
 }
 
-/**
- * @return {Boolean}
- */
-function isNewMessagesBadgeVisible() {
+function isNewMessagesBadgeVisible(): boolean {
     const hintText = Localize.translateLocal('accessibilityHints.scrollToNewestMessages');
     const badge = screen.queryByAccessibilityHint(hintText);
-    return Math.round(badge.props.style.transform[0].translateY) === -40;
+    return Math.round(badge?.props.style.transform[0].translateY) === -40;
 }
 
-/**
- * @return {Promise}
- */
-function navigateToSidebar() {
+function navigateToSidebar(): Promise<unknown> {
```

Comment:
Just FYI, I've set `Promise<void>` as a return type of `waitForBatchedUpdates`, but it's still in internal review

Reply:
`waitForBatchedUpdates` is migrated on `main`, try to merge it